### PR TITLE
feat(ext.bridge): add bridge_commands attribute

### PR DESCRIPTION
**Pull request by Middledot**
## Summary
Adds access to BridgeCommand access directly.

## Information

- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting,
      examples, ...).

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [x] If code changes were made then they have been tested.
  - [x] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why.

### CODE REVIEW

**Review by BobDotCom — CHANGES_REQUESTED** [review]
Please add a changelog entry

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ These changes are available on the `master` branch, but have not yet been releas
   order. ([#1636](https://github.com/Pycord-Development/pycord/pull/1636))
 - Support for new thread attributes `total_message_sent` and `is_pinned`.
   ([#1636](https://github.com/Pycord-Development/pycord/pull/1636))
+- Added `bridge_commands` attribute for access to bridge command objects.
```
Comment (BobDotCom) [inline CHANGELOG.md:29]:
```suggestion
- Added `bridge_commands` attribute to `ext.bridge.Bot` for access to bridge command objects.
```

**Review by BobDotCom — APPROVED** [review]

**Review by Lulalaby — APPROVED** [review]

**Comment by Middledot** [comment]
auto-merges apparently don’t work because I can’t approve

**Comment by Lulalaby** [comment]
you're the only one for this right?
@BobDotCom

**Review by Dorukyum — APPROVED** [review]

**Pull request merged**